test_scripts/Defects/5_0/2479_SDL_does_not_respond_NACK_to_second_request/2479_6_SDL_respond_ACK_from_not_protected_to_protected.lua

```diff
@@ -1,5 +1,5 @@
 ---------------------------------------------------------------------------------------------------
--- User story: https://github.com/smartdevicelink/sdl_core/issues/2379
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2479
 --
 -- Description:
 -- 1) SDL does not respond ACK on second service.
```

Comment:
Please update the description according to an actual sequence in the script(SDL does respond with ACK)

Reply:
@GetmanetsIrina Description is updated in cf6f03e

Reply:
@dboltovskyi, thank you for the update.